src/illuminatio/test_case.py

```diff
@@ -1,10 +1,15 @@
+"""
+file containing utilities for NetworkTestCases
```

Comment:
Start with a capital letter to match the rest of the docstrings (see: https://www.python.org/dev/peps/pep-0257/)